README.md

```diff
@@ -466,8 +466,56 @@ Avoid including a key without a meaningful value:
 
 ### ✅
 
+The error object needs to have following: 
+ - Be consistent
+ - Have all required info
+ - Easy parsable
+ - Should be possible to build a solid UI on top, guiding the user what happened, and how to move on
+
+```
+{
```

Comment:
Can you make sure all indentations are 4 spaces? Then it's aligned with the rest of the code snippets.

Reply:
maybe its me, but what do you mean here? can you do a example

Reply:
I fixed the indentations locally.
Will push to this PR once I get access rights from @Casperhr  :pray: